.eslintrc.json

```diff
@@ -13,7 +13,9 @@
     "no-shadow": "off",
     "no-param-reassign": "off",
     "eol-last": "off",
-    "arrow-parens": "off"
+    "arrow-parens": "off",
+    "import/extensions": "off",
+    "import/no-unresolved": "off"
```

Comment:
Please do not make any changes to the [config file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

Reply:
I put those lines initially there, I forgot to remove it. Thanks for pointing out the issue and for your feedback on the project.

Jaspreet Singh